spk/jellyfin/Makefile

```diff
@@ -12,11 +12,11 @@ DEPENDS = cross/jellyfin cross/jellyfin-web
 # x64 and armv8 archs are supported only.
 UNSUPPORTED_ARCHS = $(32bit_ARCHS)
 
```

Comment:
please add 
```
REQUIRED_MIN_DSM = 7.2
```

This is required, since node js 20 is not available for DSM < 7.2
_As a side effect the github build action will fail and packages have to be built elsewhere (i.e. locally)._

Reply:
Already solved